ams/server/src/main/java/com/netease/arctic/server/optimizing/maintainer/MaintainStrategy.java

```diff
@@ -0,0 +1,19 @@
+package com.netease.arctic.server.optimizing.maintainer;
+
+import com.netease.arctic.server.table.DataExpirationConfig;
+import org.apache.iceberg.StructLike;
+import org.apache.iceberg.expressions.Expression;
+
+import java.util.List;
+import java.util.Map;
+
+public interface MaintainStrategy {
```

Comment:
The MixedTableMaintainer has ChangeTableMaintainer and BaseTableMaintainer, Can their replace MaintainStrategy?

Reply:
It may be more complicated.
- If we scan expired files in `MixedTableMaintainer`, `MixedTableMaintainer` have to extend `IcebergTableMaintainer`;
- We need to scan change table and base table in one stage to find the latest updated file in the partition/table. Therefore I donot know how to scan both change and base table in each table maintainer.
  Do you have any good idea to achive this?

Reply:
I got you , May be the parts that cannot be handled separately are processed in MixedTableManitainer, while the parts that can be handled separately are placed in MixedTableManitainer#ChangeTableManitainer and MixedTableManitainer#BaseTableManitainer.